### PR TITLE
Fix metadata metric type in Serializer.java

### LINKED ISSUE #5693: Prometheus exporter produces invalid Prometheus text format
**Describe the bug**

The Prometheus exporter automatically adds `target_info `and `otel_scope_info` metadata metrics. Those metrics are gauges, but in their `#TYPE` line, the type is set to `info`, which is not a valid prometheus type.

As such, the export is not compliant to the spec and will be rejected by some scrappers.

**What did you expect to see?**
```
# TYPE target gauge
# HELP target Target metadata
target_info{*** SOME LABELS***} 1
# TYPE otel_scope_info gauge
# HELP otel_scope_info Scope metadata
otel_scope_info{*** SOME LABELS***} 1
```

**What did you see instead?**
```
# TYPE target info
# HELP target Target metadata
target_info{*** SOME LABELS***} 1
# TYPE otel_scope_info info
# HELP otel_scope_info Scope metadata
otel_scope_info{*** SOME LABELS***} 1
```

**What version and what artifacts are you using?**

Affects latest tag (1.28.0) and main branch.

### PR DESCRIPTION
**Pull request by lchdev**
Fix hardcoded prometheus type.

Fixes #5693

### CODE REVIEW

**Review by breedx-splk — APPROVED** [review]
Looks good to me. Can we get unit test coverage added to prevent regressions? Thanks!

**Comment by lchdev** [comment]
I actually missed the tests that were already covering the Serializer. 
Should be ok now.

**Comment by lchdev** [comment]
Hi @breedx-splk, do you expect anything more from me before the request can be merged ?

File: exporters/prometheus/src/test/java/io/opentelemetry/exporter/prometheus/SerializerTest.java
Comment (dashpole) [inline exporters/prometheus/src/test/java/io/opentelemetry/exporter/prometheus/SerializerTest.java:129]: Can we keep the info typing on OpenMetrics format?

**Comment by breedx-splk** [comment]
> Hi @breedx-splk, do you expect anything more from me before the request can be merged ?

Actually, I approved it, but 1) I'm not an approver in the java core repo, and 2) it looks like this PR isn't going against upstream...I didn't realize that when I reviewed it! Derp.

You should probably recreate it pointing at upstream if that was your intention...

**Pull request closed without merging**
